shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/ManagedBridgeSyncService.java

```diff
@@ -0,0 +1,49 @@
+package com.redhat.service.smartevents.shard.operator.v2;
+
+import com.redhat.service.smartevents.infra.v2.api.models.dto.BridgeDTO;
+import com.redhat.service.smartevents.shard.operator.v2.converters.ManagedBridgeConverter;
+import com.redhat.service.smartevents.shard.operator.v2.providers.NamespaceProvider;
+import com.redhat.service.smartevents.shard.operator.v2.resources.ManagedBridge;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.util.List;
+
+@ApplicationScoped
+public class ManagedBridgeSyncService {
```

Comment:
@vaibhavjainwiz We need a unit test for this class please.

Reply:
done